### PR TITLE
ipv6: add the debugging info for showing IPv6 header.

### PR DESCRIPTION
**Pull request by andrewhit**

### CODE REVIEW

File: src/ipv6/ipv6.c [hunk 1]
```diff
@@ -75,6 +75,22 @@ static RTE_DEFINE_PER_LCORE(struct inet_stats, ip6_stats);
         this_ip6_stats.__f__##octets += (val); \
     } while (0)
 
+#ifdef CONFIG_DPVS_MBUF_DEBUG
+static inline void ip6_show_hdr(const char *func, struct rte_mbuf *mbuf)
```
Comment (mscbg) [inline src/ipv6/ipv6.c]: CONFIG_DPVS_MBUF_DEBUG is not a good name, because we didn't dump mbuf but just ipv4/ipv6 header

Comment (andrewhit) [reply src/ipv6/ipv6.c]: How about using "CONFIG_DPVS_IP_HEADER_DEBUG"?

Comment (mscbg) [reply src/ipv6/ipv6.c]: I think it's fine.

Comment (andrewhit) [reply src/ipv6/ipv6.c]: Done.

**Pull request merged**